alembic/versions/852abaee3a25_add_email_notifications_to_user_table.py

```diff
@@ -0,0 +1,27 @@
+"""Add email_notifications to user table
+
+Revision ID: 852abaee3a25
+Revises: 0e88581a5e5b
+Create Date: 2017-04-27 18:34:51.728465
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '852abaee3a25'
+down_revision = '0e88581a5e5b'
+
+from alembic import op
+import sqlalchemy as sa
+
+from compair.models import convention
+
+
+def upgrade():
+    op.add_column('user', sa.Column('email_notifications', sa.Boolean(name='email_notifications'),
```

Comment:
maybe change this to emun. we could have more types of the notifications,e.g. disable, enable, digest, etc.